docs/developer-docs/daos/sns/tokenomics/tokenomics-intro.mdx

```diff
@@ -72,9 +72,10 @@ For the initial token allocation, i.e., defining which groups/accounts should re
 * **Seed funders**: distribution to funders (if you choose to have them) who invested in the project prior to the launch of the DAO.
 * **Funding development team**: developers who created the initial version of the DAO.
 
-To emphasize the decentralization of the DAO from the start, developers can send a strong signal by:
-* Allocating a significant part of the tokens to the DAO treasury, allowing the treasury to incentivize and reward users over time.
-* Ensure that the amount of tokens allocated to the decentralization swap is bigger than the amount allocated to seed funders and the funding development team.
+To facilitate a healthy DAO from the start, developers should:

```

Comment:
Maybe we can phrase it a bit less as a recommendation (as it is in the end up to the NNS) and more as a "this is a good practice / recommended"? 

E.g. 
To facilitate a healthy DAO from the start, it is recommended that developers:

Reply:
The first bullet is enforced, so this is not just a recommendation

Reply:
I've added a sub-bullet to the second bullet to mention that this is currently just a recommendation.